pkg/upgrade/upgrade.go

```diff
@@ -104,6 +104,16 @@ func ManagedInstance(ctx context.Context, client client.Client, jaeger v1.Jaeger
 		return jaeger, nil
 	}
 
+	if currentSemVersion.GreaterThan(latestSemVersion) {
+		// This jaeger instance has a version greater than the latest know version of the operator
+		jaeger.Logger().WithFields(log.Fields{
+			"instance":  jaeger.Name,
+			"namespace": jaeger.Namespace,
+			"to":        latestVersion,
+		}).Warn("Jaeger instance has a version greater that the latest know version")
```

Comment:
`latest version of the operator`

Reply:
latest version of the operator is a confusing message IMHO.

Is not the version of the operator, but the latest jaeger version this operator can handle. Anyway I'm not good writing good messages, so I'm gonna trust you and will change it :)